src/main/java/org/jabref/gui/fieldeditors/FileListEditorTransferHandler.java

```diff
@@ -83,20 +83,18 @@ public boolean importData(JComponent comp, Transferable t) {
             // This flavor is used for dragged file links in Windows:
             if (t.isDataFlavorSupported(DataFlavor.javaFileListFlavor)) {
                 @SuppressWarnings("unchecked")
-                List<Path> transferedFiles = (List<Path>) t.getTransferData(DataFlavor.javaFileListFlavor);
-                files.addAll(transferedFiles);
-            }
-
-            if (t.isDataFlavorSupported(urlFlavor)) {
+                List<File> transferedFiles = (List<File>) t.getTransferData(DataFlavor.javaFileListFlavor);
+                files.addAll(transferedFiles.stream().map(file -> file.toPath()).collect(Collectors.toList()));
```

Comment:
And mabye add a comment that it has to be a File object and not a path. I remember that I fixed this before the regression ;)